vscode-lean4/src/infoview.ts

```diff
@@ -399,13 +417,14 @@ export class InfoProvider implements Disposable {
             // inside the webview through the standard message event.
             // The receiving of these messages is done inside webview\index.ts where it
             // calls window.addEventListener('message',...
-            webviewPanel.rpc = new Rpc(m => webviewPanel.webview.postMessage(m));
+            webviewPanel.rpc = new Rpc(m => { if (!closed) void webviewPanel.webview.postMessage(m) });
             webviewPanel.rpc.register(this.editorApi);
 
             // Similarly, we can received data from the webview by listening to onDidReceiveMessage.
-            webviewPanel.webview.onDidReceiveMessage(m => webviewPanel.rpc.messageReceived(m))
+            webviewPanel.webview.onDidReceiveMessage(m => { if (!closed) webviewPanel.rpc.messageReceived(m) })
             webviewPanel.api = webviewPanel.rpc.getApi();
             webviewPanel.onDidDispose(() => {
+                closed = true;
```

Comment:
I am not familiar with how `let` is scoped in JS. Are you sure setting this will update the state of the closures above? A class member might be clearer.

Reply:
Sure that works, I pushed the change.